fe/fe-core/src/main/java/org/apache/doris/http/rest/BootstrapFinishAction.java

```diff
@@ -37,6 +37,7 @@
  * {"status":"OK","msg":"Success","replayedJournal"=123456, "queryPort"=9000, "rpcPort"=9001}
  * {"status":"FAILED","msg":"err info..."}
  */
+@Deprecated
```

Comment:
Why deprecate this?

Reply:
> Why deprecate this?

get heartbeat info by rpc method is a more stable way, once get heartbeat by thrift, BootstrapFinishAction is not needed anymore, here we just keep compatible when fe upgrade

Reply:
I thought that `BootstrapFinishAction` is nothing to do with the heartbeat?
It is used for checking whether the FE is started successfully.

Reply:
> I thought that `BootstrapFinishAction` is nothing to do with the heartbeat? It is used for checking whether the FE is started successfully.

 controller.registerHandler(HttpMethod.GET, "/api/bootstrap", new BootstrapFinishAction(controller))
it is now used for heartbeat

   private HeartbeatResponse getHeartbeatResponseByHttp() {
            String url = "http://" + fe.getHost() + ":" + Config.http_port
                    + "/api/bootstrap?cluster_id=" + clusterId + "&token=" + token;

Reply:
Oh, I forgot. But this API is also used for checking whether FE is started.
So we still need it. Just not use it for heartbeat.

Reply:
And by the way, most of user does not switch the heartbeat method to thrift way. So maybe we need notice the user by somehow.

Reply:
Ok, I see